logserver/jwt.php

```diff
@@ -0,0 +1,74 @@
+<?php
+// SPDX-FileCopyrightText: 2009 Fermi Research Alliance, LLC
+// SPDX-License-Identifier: Apache-2.0
+
+/*  Script to test JWT generation and verification in PHP:
+    php jwt.php // To run self test
+    php jwt.php TOKEN  // To decode the token
+    php jwt.php PAYLOAD TOKEN  // To encode PAYLOAD and save it to TOKEN
+    Uses the configuration in logging_config.json:
+    - secret_key_path (/var/lib/gwms-factory/server-credentials/jwt_secret.key)
+    - secret_key (VG9rZVNlY3JldEtleQo=)
+    - verbose (True)  // Print verbose output
+    - debug (False)  // Print debug output
+*/
+// declare(strict_types=1);
+use Firebase\JWT\JWT;
+use Firebase\JWT\Key;
+require_once('./vendor/autoload.php');
+
+// Hardcoded parameters
+$configFile = 'logging_config.json';
+// Read and extract configuration
+$config = json_decode(file_get_contents($configFile), true);
+if (! $config) {
+    trigger_error("Empty config ($configFile) - the file may be missing or be invalid JSON", E_USER_WARNING);
+}
+$secretKeyFile = $config['secret_key_path'] ?? '/var/lib/gwms-factory/server-credentials/jwt_secret.key';
+$defaultSecretKey = $config['secret_key'] ?? 'VG9rZVNlY3JldEtleQo=';
```

Comment:
Is it okay/safe to have this secret key defined here (the actual key itself)?

Reply:
removed and protected from empty string